tests/testthat/test-minimal_example.R

```diff
@@ -22,19 +29,41 @@ row_groups <- readr::read_tsv(paste0(dir, "row_groups.tsv"))
 palettes <- jsonlite::read_json(paste0(dir, "palettes.json"), simplifyVector = TRUE)
 palettes$pie <- unlist(palettes$pie)
 
+if(DEBUG){
+  column_info <- column_info %>% 
+    mutate(directory = if_else(is.na(directory), NA, paste0(image_dir_test, "test_data")) )
+  data <- data %>%
+    mutate(image_full = paste0(image_dir_test, image_full))
+}
+
 legends <- list(
+  list(
+    title = "Bar",
+    palette = "bar",
+    enabled = TRUE,
+    geom = "bar"
+  ),
   list(
     title = "Text",
     palette = "black_text",
     enabled = TRUE,
     geom = "text",
     labels = c("propA", "propB", "propC"),
-    values = c("Property of A", "Property of B", "Property of C")
+    values = c("A", "B", "C")
+  ),
+  list(
+    title = "Image",
+    palette = "black_text",
+    enabled = TRUE,
+    geom = "image",
+    size = 3.88,
+    labels = c(paste0(image_dir_test, "test_data/one.png"), paste0(image_dir_test, "test_data/two.png"), paste0(image_dir_test, "test_data/three.png")),
+    values = c("One", "Two", "Three")
   )
 )
 
 test_that("minimal example funky_heatmap works", {
-  g <- funky_heatmap(
+  g <- funkyheatmap::funky_heatmap(
```

Comment:
Undo revert of fix in #42

```suggestion
  g <- funkyheatmap::funky_heatmap(
```